kasa/cli.py

```diff
@@ -135,6 +151,18 @@ def _device_to_serializable(val: SmartDevice):
     is_flag=True,
     help="Output raw device response as JSON.",
 )
+@click.option(
+    "--encrypt-type",
+    envvar="KASA_ENCRYPT_TYPE",
+    default=EncryptType.Xor.value,
+    type=click.Choice(ENCRYPT_TYPES, case_sensitive=False),
+)
+@click.option(
+    "--tplink-device-type",
+    envvar="KASA_TPLINK_DEVICE_TYPE",
```

Comment:
```suggestion
    envvar="KASA_DEVICE_TYPE",
```
We want to keep the envvars consistent (see #570) even when they are not currently documented anywhere outside the source code :-)

Reply:
So we already have a `device_type` which is the `DeviceType` string, `stripsocket`, `dimmer` etc.  This parameter is for the device_type as TPlink reports it, i.e. `IOT.SMARTPLUGSWITCH`, `SMART.KASAPLUG etc`

Reply:
Gotcha, I'd still like to avoid naming it "TPLINK_DEVICE_TYPE" as it's not really descriptive for what it really is.

Reply:
How about KASA_DEVICE_FAMILY?

Reply:
Sounds good to me.

Reply:
Done